src/media/css/buttons.styl

```diff
@@ -188,6 +188,10 @@ a.button {
             display: inline-block;
         }
     }
+    +rtl() {
```

Comment:
I feel like these `+rtl()` blocks should go immediately after the styles for this element. L152 in this case.